kernel/api/src/main/java/org/sakaiproject/site/api/SiteService.java

```diff
@@ -297,6 +297,12 @@ public boolean isIgnoreSoftlyDeleted()
 
 		/** Get any deleted sites, normally used by admin or purge job. */
 		public static final SelectionType ANY_DELETED = new SelectionType("anyDeleted", false, false, false, false);
+
+		/** Get published sites the current user has access to */
+		public static final SelectionType ACTIVE = new SelectionType("active", true, true, true, true);
+
+		/** Get unpublished sites the current user has access to */
+		public static final SelectionType INACTIVE = new SelectionType("inactive", true, true, false, true);
```

Comment:
Both of these SelectionTypes duplicate ones above: ACTIVE duplicates PUBVIEW and INACTIVE duplicates MEMBER